### PR TITLE
fix(tooltip): cleanup containers properly

### PR DESCRIPTION
**Pull request by 0x0dr1y**
This fixes an issue that was kinda teased in https://github.com/mtorromeo/vue-patternfly3/issues/37.

After a bit investigation it turned out that the app containers were never destroyed when unmounting / updating the directive. With many updates this will lead to quite some abandoned DOM nodes as you can imagine.

The reason for that is that the check in the unbind hook will fail, because the data of the vm cannot be accessed.

```
const container = (vm as any)?.container;
if (container instanceof HTMLElement) {
 document.documentElement.removeChild(container);
}
```

This happens because `createApp` will create an application instance that **doesn't** hold data. The data will only be available in the component public instance which is created by mounting the app instance (`app.mount()`) .

So basically we could either pass around the public instance (not worth it imo), or pass the container in the el as done within this PR.

### CODE REVIEW

**Comment by mtorromeo** [comment]
Nicely done, thanks!
